### DISCUSSION TITLE
models/197/DF-ODENet_DNNmodel

### DISCUSSION THREAD

**Discussion by giscus** [deepmodeling/AIS-Square#234] [Announcements]
# models/197/DF-ODENet_DNNmodel

https://www.aissquare.com/models/detail?pageType=models&name=DF-ODENet_DNNmodel&id=197

**Comment by WUCHIL** [comment]
作者您好，请问这里面训练神经网络的数据采样方法一直令我困惑不解，请问能否公开相关数据采样代码，感激不尽
